tests/integrated/test-boutcore/slicing/notworking

```diff
@@ -0,0 +1,9 @@
+x[:, np.newaxis, :, :].shape
+x[:, None, :, :].shape
+x[:, np.newaxis] + x[np.newaxis, :]
+x[~np.isnan(x)]
+x[x < 0]
+x[x < 0] += 20
+x[2:7] = 1
+x[2:7] = np.arange(5)
+x[np.array([1, 1, 3, 1])] += 1
```

Comment:
Are these examples of slices that won't work? Could you instead add a `README.md` to this directory and list them there?